source/AI.cpp

```diff
@@ -2962,13 +2962,16 @@ void AI::AutoFire(const Ship &ship, FireCommand &command, bool secondary) const
 	bool beFrugal = (ship.IsYours() && !escortsUseAmmo);
 	if(person.IsFrugal() || (ship.IsYours() && escortsAreFrugal && escortsUseAmmo))
 	{
-		// Frugal ships only expend ammunition if they have lost 50% of shields
-		// or hull, or if they are outgunned.
+		// Frugal ships do not expend ammunition except if they have lost 50%
+		// of shields and hull combined, or if they are outgunned.
 		beFrugal = (ship.Hull() + ship.Shields() > 1.5);
```

Comment:
This isn't 50% of hull and shields combined.
At best, it is 25% of total health, but that assumes hull and shields are equal.
Perhaps `the sum of the fractions of its hull and shield remaining is below 1.5`.
I'm not entirely sure that the comment actually needs changing. It is difficult to come up with an actually better description of what is happening.

Reply:
Yeah thats why it confused me in the first place...

Reply:
How about we just say "a significant portion of hull and shields was lost"?

Reply:
Sure